modules/apps/transfer/ibc_module.go

```diff
@@ -89,16 +90,21 @@ func (im IBCModule) OnChanOpenInit(
 		version = types.Version
 	}
 
-	if version != types.Version {
-		return "", errorsmod.Wrapf(types.ErrInvalidVersion, "expected %s, got %s", types.Version, version)
+	if !slices.Contains(types.SupportedVersions, version) {
+		return "", errorsmod.Wrapf(types.ErrInvalidVersion, "invalid version: expected %s or %s, got %s", types.Version1, types.Version, version)
```

Comment:
In channel upgrades I remember we used a string like...

```suggestion
		return "", errorsmod.Wrapf(types.ErrInvalidVersion, "expected one of [%s, %s], got %s", types.Version1, types.Version, version)
```